client/pages/styles/index.css

```diff
@@ -95,7 +98,7 @@ h1 {
 }
 
 .menu a:hover {
-  color: #B9946A;
+  color: #b9946a;
```

Comment:
Can we use the root color variable here? `var(--wood)` This actually applies to all colors we used in index.css, so let's plan to update these colors in the upcoming PRs!